core/sdk-core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -138,6 +142,16 @@ protected RequestOverrideConfiguration(Builder<?> builder) {
         return metricPublishers;
     }
 
+    /**
+     * Returns the additional execution attributes to be added to this request.
+     * This collection of attributes is added in addition to the attributes set on the client.
+     * An attribute value added on the client within the collection of attributes is superseded by an
+     * attribute value added on the request.
+     */
+    public ExecutionAttributes executionAttributes() {
+        return executionAttributes;
```

Comment:
`RequestOverrideConfiguration` should be immutable; can we create an immutable view of `executionAttributes` to return?

Reply:
Like a UnmodifiableMap?